.github/workflows/ci.yaml

```diff
@@ -102,6 +104,10 @@ jobs:
         with:
           name: release-packages
           path: ./release-packages
+      - name: Remove docker images
+        run: |
+          docker rm $(docker ps -aq)
+          docker rmi $(docker images -q)
```

Comment:
Might be cleaner to just use `docker system prune` <https://docs.docker.com/engine/reference/commandline/system_prune/>

Reply:
TIL! Thanks for the tip @vapurrmaid 🙌